pkg/provisioner/crdprovisioner.go

```diff
@@ -158,6 +159,10 @@ func (c *CrdProvisioner) CreateVolume(
 			return false, err
 		}
 		if azVolumeCreated.Status != nil {
+			if azVolume.Status.Error != nil {
```

Comment:
Need these error handling in `DeleteVolume`, `PublishVolume`, `UnpublishVolume` and `ExpandVolume` as well

Reply:
Done